content/help/ops/security/root-transition/index.md

```diff
@@ -0,0 +1,199 @@
+---
+title: Extending the lifetime of the Istio self-signed root certificate
```

Comment:
I think this should be quite a bit shorter. Titles show up in the left navigation area, so there's limited space. For example:

Self-Signed Certificate Lifetime

Reply:
Sure.
Changed to: Extending Self-Signed Certificate Lifetime